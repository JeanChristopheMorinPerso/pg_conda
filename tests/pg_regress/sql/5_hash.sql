-- Test the hash function
BEGIN;

CREATE EXTENSION pg_conda;

SELECT DISTINCT v FROM (VALUES 
    ('1.1.0'::condaversion),
    ('1.01.0'::condaversion),
    ('2.0.4'::condaversion),
    ('1.0.0'::condaversion)
) v(v);

ROLLBACK;
