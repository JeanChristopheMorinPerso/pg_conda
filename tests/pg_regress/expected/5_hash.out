-- Test the hash function
BEGIN;
CREATE EXTENSION pg_conda;
SELECT DISTINCT v FROM (VALUES 
    ('1.1.0'::condaversion),
    ('1.01.0'::condaversion),
    ('2.0.4'::condaversion),
    ('1.0.0'::condaversion)
) v(v);
   v   
-------
 1.0.0
 2.0.4
 1.1.0
(3 rows)

ROLLBACK;
